eng/common/TestResources/New-TestResources.ps1

```diff
@@ -164,6 +168,9 @@ if ($ProvisionerApplicationId) {
         $null = Disconnect-AzAccount -AzureContext $provisionerAccount.Context
     }
 }
+# Set standard environment variables from configuration
```

Comment:
@pakrym -- I'm setting a default `AZURE_AUTHORITY_HOST` value here because Identity doesn't support scoped variable names like `SERVICENAME_AZURE_AUTHORITY_HOST`. I suspect other values from the context's environment will become necessary at a global level as well though I'm not aware of any use cases for those yet.

Reply:
The point of having prefixed variables was to be able to have multiple sets of test resources working at once.

We have code that reads these variables and configures credential types.

Can you please add this value to https://github.com/Azure/azure-sdk-for-js/blob/45af11938248e5b00c870c025564b9b46ba9402c/eng/common/TestResources/New-TestResources.ps1#L307

so we can use it from C# test environment implementation and so it flows into test environment file.

It's also concerning that now we have 2 sets of outputs from the script depending on how it's being used.

Reply:
And JS test create credentials manually and pass the values too: https://github.com/Azure/azure-sdk-for-js/blob/4f31bc9b1089ee5b7de404437bdc46e9d2634d57/sdk/keyvault/keyvault-secrets/test/utils/testAuthentication.ts#L29

Reply:
I can see a valid argument for setting the global `AZURE_AUTHORITY_HOST` environment variable at another layer (not `New-TestResources.ps1`) and keeping the environment variables scoped to the service being deployed here.